src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.styles.js

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from 'styled-components';
+import colors from '../../styles/colors';
+
+export const Dash = styled.span`
+  font-size: 16px;
+  color: #d4bddb;
```

Comment:
This color is not listed in colors file? Perhaps you can add it.

Reply:
Added